mods/ra/maps/survival02/survival02.lua

```diff
@@ -397,4 +397,9 @@ WorldLoaded = function()
 	InitObjectives()
 	InitMission()
 	SetupSoviets()
+
+	Trigger.AfterDelay(0, function()
+		-- The turret most to the right will attack an AA gun
```

Comment:
This is just inside the test commit, and doesn't matter thus.
Thanks though. :)